ReducingCrime.Rmd

```diff
@@ -5,8 +5,12 @@ author: "Harith Elrufaie and Gaurav Desai"
 output: pdf_document
 ---
 
-# Exploratory Data Analysis
-Perform an exploratory analysis to understand the determinants of crime and to generate policy suggestions that are applicable to local government.
+#Introduction
+The goal of this report is to analyze and study dataset of crime statistics for a selection of counties in North Carolina to help a political campaign to understand the determinants of crime. The outcome of this study is policy suggestions that are applicable to local government.
+
+#Exploratory Data Analysis
```

Comment:
This is misplaced, should come after Removal of NA, just before Univariate analysis begins